apps/meteor/tests/end-to-end/api/24-methods.js

```diff
@@ -2067,6 +2069,72 @@ describe('Meteor.methods', function () {
 				.end(done);
 		});
 
+		it('should add a quote attachment to a message', async () => {
+			const siteUrl = process.env.SITE_URL || process.env.TEST_API_URL || 'http://localhost:3000';
+
+			const quotedMsgLink = `${siteUrl}/group/${roomName}?msg=${messageWithMarkdownId}`;
+			await request
+				.post(methodCall('updateMessage'))
+				.set(credentials)
+				.send({
+					message: JSON.stringify({
+						method: 'updateMessage',
+						params: [{ _id: messageId, rid, msg: `${quotedMsgLink} updated` }],
+						id: 'id',
+						msg: 'method',
+					}),
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.a.property('success', true);
+					expect(res.body).to.have.a.property('message').that.is.a('string');
+				});
+
+			await request
+				.get(api(`chat.getMessage?msgId=${messageId}`))
+				.set(credentials)
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.property('message').that.is.an('object');
+					expect(res.body.message).to.have.property('msg', `${quotedMsgLink} updated`);
+					expect(res.body.message).to.have.property('attachments').that.is.an('array').that.has.lengthOf(1);
+					expect(res.body.message.attachments[0]).to.have.property('message_link').that.is.a('string');
+				});
+		});
+
+		it('should remove a quote attachment from a message', async () => {
```

Comment:
Can we add a test case for when we are updating or replacing or both the existing quotes of a message?